sheet_test.go

```diff
@@ -463,3 +465,21 @@ func TestAttrValToFloat(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, 42.1, got)
 }
+
+func TestSetSheetBackgroundFromBytes(t *testing.T) {
+	files := []string{"../../excelize.svg", "excel.emf", "excel.emz", "excel.gif", "excel.jpg", "excel.png", "excel.tif", "excel.wmf", "excel.wmz"}
+	f, err := OpenFile("./test/EmptyWorkbook.xlsx")
```

Comment:
The workbook `test/EmptyWorkbook.xlsx` seem to be modified after the test, we need to keep it as the origin.

Reply:
The test will set the background image for `f`, but instead of calling `Save` it calls `SaveAs` to save another file, 
rather than itself. I checked before calling test and after calling test, the size of `./test/EmptyWorkbook.xlsx` is 8476 bytes, and the content is also consistent.

Reply:
The test will set the background image for f, but instead of calling Save it calls SaveAs to save another file,
rather than itself. I checked before calling test and after calling test, the size of ./test/EmptyWorkbook.xlsx is 8454 bytes, and the content is also consistent.

Reply:
All right. Could we use the `NewFile` function to prepare a blank workbook instead of adding a new file in the test folder?

Reply:
Resolved